examples/fasthtml/api/index.py

```diff
@@ -0,0 +1,12 @@
+from fasthtml.common import *
```

Comment:
This file will be removed in a follow-up (it's a duplicate of `main.py`). See the comment in the `vercel.json` file below.